examples/limit-clause/limit_clause.description

```diff
@@ -1 +1 @@
-// The `limit` clause limits the number of results from the earlier clauses.
+// The `limit` clause limits the number of results from the query.
```

Comment:
Is this change correct? A query expression can have multiple limit clauses? @gimantha

Reply:
limits the frames (results) from the query.

Reply:
Limits them for the next clause, right? Not necessarily for the entire query expression?

Reply:
I got it. There can be multiple limit clauses.